check-updates.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
Note https://github.com/flathub/flatpak-external-data-checker/ will be opening PRs automatically if URL and checksum combination becomes invalid, but probably won't update the version in appdata.